Library/Newsletter.swift

```diff
@@ -62,6 +55,8 @@ public enum Newsletter {
       return Strings.Welcome_to_our_library_Peruse_the_stacks_with_us()
     case .alumni:
       return Strings.profile_settings_newsletter_alumni_newsletter()
+    case .music:
+      return "It's like the radio but nothing sucks and also it’s a newsletter."
```

Comment:
I'd recommend putting this in a `localizedString()` function and use the expected `key` and `defaultValue`.  That way we can even merge this PR before the translations are done and later just run `make strings` and merge that in. Also, we can easily search the codebase for `localizedString` to know where it needs to be replaced in the event that we forget where we didn't use localization.

tl;dr I think it's a safer practice to use `localizedString(key:defaultValue:)` than just a raw string 👍

Reply:
lol I probably should've put the tl;dr at the top 😬

Reply:
lol I will change that